README.md

```diff
@@ -13,7 +13,8 @@
 <div>
 <img align="left" src="https://raw.githubusercontent.com/inkremental/anvil/master/logo/ic_launcher.png" alt="logo" width="96px" height="96px" />
 <p>
-Anvil is a small Java library for creating reactive user interfaces. Originally inspired by <a href="https://facebook.github.io/react/">React</a>, it suits well as a view layer for <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel">MVVM</a> or <a href="http://redux.js.org/">Redux</a> design patterns.
+Inkremental is a small library for creating reactive user interfaces. Originally inspired by <a href="https://facebook.github.io/react/">React</a>, it suits well as a view layer for <a href="https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel">MVVM</a> or <a href="http://redux.js.org/">Redux</a> design patterns.
+It is a fork of Java library Anvil, fully rewritten in Kotlin.
```

Comment:
I guess we can word that like "Started as a fork of Java library Anvil, it is fully rewriten in Kotlin, with many improvements pulled in and more to come" or something like that to make sure we highlight further movement in our own direction.